### PR TITLE
Revert CC-174

### PR DESCRIPTION
**Pull request by jgeorge82**
giphy-3

This reverts CC-174 and updates the version numbering so that we don't accidentally re-introduce these issues.

### CODE REVIEW

**Review by brbowden — APPROVED** [review]
👍 Looks good

File: LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj [hunk 1]
```diff
@@ -46,20 +46,24 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
+    <Reference Include="Java.Interop" />
     <Reference Include="GoogleGson">
-      <HintPath>\tmp\nuget\GoogleGson.2.8.1\lib\MonoAndroid\GoogleGson.dll</HintPath>
+      <HintPath>..\packages\GoogleGson.2.8.1\lib\MonoAndroid\GoogleGson.dll</HintPath>
```
Comment (brbowden) [inline LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj]: Sorry, just noticed this as I was approving - can we change these back to `\tmp\nuget` path references? That helps a little with not duplicating the same nuget packages in multiple locations across multiple repos

**Review by joe-ifit — APPROVED** [review]
Looks good

File: LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj (hunk 1 above)
Comment (chrisr-ifit) [reply LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj]: ^^

**Review by chrisr-ifit — CHANGES_REQUESTED** [review]

**Review by chrisr-ifit — APPROVED** [review]

**Review by brbowden — APPROVED** [review]

**Pull request merged**